examples/src/test/java/jdocs/cassandra/WordCountDocExample.java

```diff
@@ -229,5 +245,329 @@ public WordCountHandler(ProjectionId projectionId, WordCountRepository repositor
     //#loadingOnDemand
   }
 
+  interface IllstrateActorLoadingInitialState {
+
+    //#actorHandler
+    class WordCountActorHandler extends ActorHandler<WordEnvelope, WordCountProcessor.Command> {
+      private final ActorSystem<?> system;
+      private final Duration askTimeout = Duration.ofSeconds(5);
+
+      WordCountActorHandler(Behavior<WordCountProcessor.Command> behavior, ActorSystem<?> system) {
+        super(behavior);
+        this.system = system;
+      }
+
+      @Override
+      public CompletionStage<Done> process(ActorRef<WordCountProcessor.Command> actor, WordEnvelope envelope) {
+        CompletionStage<WordCountProcessor.Result> result =
+            AskPattern.ask(actor, (ActorRef<WordCountProcessor.Result> replyTo) -> new WordCountProcessor.Handle(envelope, replyTo),
+                askTimeout, system.scheduler());
+
+        return result
+            .thenCompose(r -> {
+              if (r.error.isPresent()) {
+                CompletableFuture<Done> err = new CompletableFuture<>();
+                err.completeExceptionally(r.error.get());
+                return err;
+              } else {
+                return CompletableFuture.completedFuture(Done.getInstance());
+              }
+            });
+      }
+    }
+    //#actorHandler
+
+    //#behaviorLoadingInitialState
+    public class WordCountProcessor {
+      public interface Command {
+      }
+
+      public static class Handle implements Command {
+        public final WordEnvelope envelope;
+        public final ActorRef<Result> replyTo;
+
+        public Handle(WordEnvelope envelope, ActorRef<Result> replyTo) {
+          this.envelope = envelope;
+          this.replyTo = replyTo;
+        }
+      }
+
+      public static class Result {
+        public final Optional<Throwable> error;
+
+        public Result(Optional<Throwable> error) {
+          this.error = error;
+        }
+      }
```

Comment:
I find it strange to have an `Optional<Throwable>`, but we will be able to improve it when we get the `askWithStatus` merged.